apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-854-Fix-PartD-Enrollment-Smaller-Transactions/05_process_ccw_table.sql

```diff
@@ -0,0 +1,387 @@
+            rcd.BENE_ID,
+            Aug1,
+            rcd.FIPS_STATE_CNTY_AUG_CD,
+            rcd.MDCR_STUS_AUG_CD,
+            rcd.MDCR_ENTLMT_BUYIN_8_IND,
+            rcd.HMO_8_IND,
+            rcd.PTC_CNTRCT_AUG_ID,
+            rcd.PTC_PBP_AUG_ID,
+            rcd.PTC_PLAN_TYPE_AUG_CD,
+            rcd.PTD_CNTRCT_AUG_ID,
+            rcd.PTD_PBP_AUG_ID,
+            rcd.PTD_SGMT_AUG_ID,
+            rcd.RDS_AUG_IND,
+            rcd.CST_SHR_GRP_AUG_CD,
+            rcd.META_DUAL_ELGBL_STUS_AUG_CD);
+            
+        -- Sept
+        call update_bene_monthly(
+            rcd.BENE_ID,
+            Sep1,
+            rcd.FIPS_STATE_CNTY_SEPT_CD,
+            rcd.MDCR_STUS_SEPT_CD,
+            rcd.MDCR_ENTLMT_BUYIN_9_IND,
+            rcd.HMO_9_IND,
+            rcd.PTC_CNTRCT_SEPT_ID,
+            rcd.PTC_PBP_SEPT_ID,
+            rcd.PTC_PLAN_TYPE_SEPT_CD,
+            rcd.PTD_CNTRCT_SEPT_ID,
+            rcd.PTD_PBP_SEPT_ID,
+            rcd.PTD_SGMT_SEPT_ID,
+            rcd.RDS_SEPT_IND,
+            rcd.CST_SHR_GRP_SEPT_CD,
+            rcd.META_DUAL_ELGBL_STUS_SEPT_CD);
+            
+        -- Oct
+        call update_bene_monthly(
+            rcd.BENE_ID,
+            Oct1,
+            rcd.FIPS_STATE_CNTY_OCT_CD,
+            rcd.MDCR_STUS_OCT_CD,
+            rcd.MDCR_ENTLMT_BUYIN_10_IND,
+            rcd.HMO_10_IND,
+            rcd.PTC_CNTRCT_OCT_ID,
+            rcd.PTC_PBP_OCT_ID,
+            rcd.PTC_PLAN_TYPE_OCT_CD,
+            rcd.PTD_CNTRCT_OCT_ID,
+            rcd.PTD_PBP_OCT_ID,
+            rcd.PTD_SGMT_OCT_ID,
+            rcd.RDS_OCT_IND,
+            rcd.CST_SHR_GRP_OCT_CD,
+            rcd.META_DUAL_ELGBL_STUS_OCT_CD);
+            
+        -- Nov
+        call update_bene_monthly(
+            rcd.BENE_ID,
+            Nov1,
+            rcd.FIPS_STATE_CNTY_NOV_CD,
+            rcd.MDCR_STUS_NOV_CD,
+            rcd.MDCR_ENTLMT_BUYIN_11_IND,
+            rcd.HMO_11_IND,
+            rcd.PTC_CNTRCT_NOV_ID,
+            rcd.PTC_PBP_NOV_ID,
+            rcd.PTC_PLAN_TYPE_NOV_CD,
+            rcd.PTD_CNTRCT_NOV_ID,
+            rcd.PTD_PBP_NOV_ID,
+            rcd.PTD_SGMT_NOV_ID,
+            rcd.RDS_NOV_IND,
+            rcd.CST_SHR_GRP_NOV_CD,
+            rcd.META_DUAL_ELGBL_STUS_NOV_CD);
+            
+        -- Dec
+        call update_bene_monthly(
+            rcd.BENE_ID,
+            Dec1,
+            rcd.FIPS_STATE_CNTY_DEC_CD,
+            rcd.MDCR_STUS_DEC_CD,
+            rcd.MDCR_ENTLMT_BUYIN_12_IND,
+            rcd.HMO_12_IND,
+            rcd.PTC_CNTRCT_DEC_ID,
+            rcd.PTC_PBP_DEC_ID,
+            rcd.PTC_PLAN_TYPE_DEC_CD,
+            rcd.PTD_CNTRCT_DEC_ID,
+            rcd.PTD_PBP_DEC_ID,
+            rcd.PTD_SGMT_DEC_ID,
+            rcd.RDS_DEC_IND,
+            rcd.CST_SHR_GRP_DEC_CD,
+            rcd.META_DUAL_ELGBL_STUS_DEC_CD);
+            
+        rcd_cnt := rcd_cnt + 1;
+        
+        if rcd_cnt % 10000 = 0
+        THEN
+            COMMIT;
```

Comment:
Doesn't this need to be followed by another `BEGIN`? Without it, I think you're running non-transactionally.